api/app_analytics/split_testing/views.py

```diff
@@ -0,0 +1,107 @@
+from django.db.models import F, QuerySet
+from django.utils.decorators import method_decorator
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import mixins, status, viewsets
+from rest_framework.generics import CreateAPIView, ListAPIView
+from rest_framework.response import Response
+
+from environments.authentication import EnvironmentKeyAuthentication
+from environments.permissions.permissions import EnvironmentKeyPermissions
+from features.models import FeatureState
+
+from .models import ConversionEventType, SplitTest
+from .permissions import ConversionEventTypePermissions, SplitTestPermissions
+from .serializers import (
+    ConversionEventSerializer,
+    ConversionEventTypeQuerySerializer,
+    ConversionEventTypeSerializer,
+    SplitTestQuerySerializer,
+    SplitTestSerializer,
+)
+
+
+class CreateConversionEventView(CreateAPIView):
+    permission_classes = (EnvironmentKeyPermissions,)
+    authentication_classes = (EnvironmentKeyAuthentication,)
+    serializer_class = ConversionEventSerializer
+
+    def post(self, *args, **kwargs) -> Response:
+        response = super().post(*args, **kwargs)
+
+        # Return an empty client to keep options open to process
+        # this in a task one day.
+        if response.status_code == 201:
+            return Response(status=status.HTTP_202_ACCEPTED)
```

Comment:
Hmm... this is going to mean the documentation isn't correct. Probably the easiest way to fix this would be to add the `swagger_auto_schema` decorator to the `post` method here.

Reply:
Nice catch! I've added the swagger response but the format looks weird for 202s

```python
@swagger_auto_schema(request_body=no_body, responses={202: ""})
```